### PR TITLE
Create a new supplemental figure for cell death metrics

### PR DESCRIPTION
**Pull request by chantelleleveille**
# Goal:

To quantify cell health over the course of our timelapse movies

# This PR includes:

- Function to visualize cell death and cell division events normalized by number of cell in the FOV
- Function to filter the last 7 hours of the timelapse to visualize any of the results from our anlaysis workflows without that data. We found no significant differences. 
- Workflow to visualize cell death counts for new supplemental figure. 

# Result of workflow 

cb4e8ec3-533f-4ad1-85b3-eb8e5d52f318
7524b3a5-6adf-4b0c-a128-6592e6983e19
7711a406-5870-482b-80a5-a75bba809f4b

### CODE REVIEW

File: nuc_morph_analysis/analyses/cell_health/cell_health_workflow.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# %%
```
Comment (jcass11) [inline nuc_morph_analysis/analyses/cell_health/cell_health_workflow.py:1]: 🎉 great! Should we add this to the run_all_manuscript_workflows.py file too?

File: nuc_morph_analysis/analyses/lineage/plot/single_generation.py [hunk 2]
```diff
@@ -131,9 +131,15 @@ def plot_corr(
 
         if unity_line:
             plt.plot(lim, lim, color="black", linestyle="--", alpha=0.2, zorder=0)
+
```
Comment (jcass11) [inline nuc_morph_analysis/analyses/lineage/plot/single_generation.py:134]: Nice!

File: nuc_morph_analysis/analyses/cell_health/cell_health_plots.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        event_count = []
+        num_cells = []
+        
+        if event_type == 'cell_death':
+            df_event = df_colony.loc[df_colony.groupby('track_id')['index_sequence'].idxmax()]
+            event_label = 'death'
+            lim1 = (0, 27)
+            lim2 = (0, 2.5)
+            
+            df_last = df_colony.loc[df_colony.groupby('track_id')['index_sequence'].idxmax()]
```
Comment (mogres) [inline nuc_morph_analysis/analyses/cell_health/cell_health_plots.py]: This is identical to `df_event`

**Review by jcass11 — APPROVED** [review]
New workflow ran without error and figures look great!

**Review by mogres — APPROVED** [review]
Workflows ran without errors and the results look great!

File: nuc_morph_analysis/analyses/cell_health/cell_health_plots.py (hunk 3 above)
Comment (chantelleleveille) [reply nuc_morph_analysis/analyses/cell_health/cell_health_plots.py]: Oh good catch thank you! Fixing now.

File: nuc_morph_analysis/analyses/cell_health/cell_health_workflow.py (hunk 1 above)
Comment (chantelleleveille) [reply nuc_morph_analysis/analyses/cell_health/cell_health_workflow.py:1]: Yes!! On it!

**Pull request merged**